pulp_container/app/tasks/sync_stages.py

```diff
@@ -542,6 +555,12 @@ async def create_signatures(self, man_dc, signature_source):
                         "{} is not accessible, can't sync an image signature. "
                         "Error: {} {}".format(signature_url, exc.status, exc.message)
                     )
+                except PayloadTooLarge as e:
```

Comment:
Again, I believe we cannot mix `aiohttp`'s exceptions with `rest_framework`'s exceptions. We are dealing with two different stacks here.